package.json

```diff
@@ -44,12 +44,12 @@
 		"prop-types": "15.5.10",
 		"querystringify": "1.0.0",
 		"re-resizable": "4.4.8",
-		"react": "16.3.2",
+		"react": "^16.4.1",
```

Comment:
Why the change to using caret? Locking down package versions as tightly as possible is a good idea, so I think this should be changed to `16.4.1` over `^16.4.1`.